README.md

```diff
@@ -2,21 +2,24 @@
 
 Sample project with a simple implementation of the webxdc read and write APIs.
 
+
 ## Demo (no server or installation required)
 
 1. Open `index.html` in your web browser
 2. Click 'Add Peer' to open as many peers as you like
 3. Type a message and press 'Send' to see the update in each peer. (For Safari you might need to check the setting under Develop > Disable Local File Restrictions.)
 
+
 ## Developing webxdc apps
 
-Simply copy `webxdc.js` from this repo beside your `index.html` and you are ready to go.
+Simply copy `webxdc.js` from this repo beside your `index.html` and you are ready to go
```

Comment:
I think this repo should be converted in a template, so it is easy to start a new github project clicking a button and having this project structure and files already in place.

We already have some templates, check:
https://github.com/orgs/webxdc/repositories?q=&type=template&language=&sort=

Converting to template is easy, go to https://github.com/webxdc/hello/settings and mark the first checkbox "Template repository"

Reply:
ah, interesting!

Reply:
will do 👊